xml_inscriptions/transcribed/IN.Crawf.LWSR.G.17.xml

```diff
@@ -0,0 +1,169 @@
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date notBefore="0001" notAfter="0002">Date to be displayed</date>
+                            <placeName ref="europe.italy.rome">Italy, Rome, San Callisto catacomb</placeName>
+                        </origin>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#SEG_28"/>
+                        <biblScope unit="item">no. 805</biblScope>
+                    </bibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#ZPE_28"/>
+                        <biblScope unit="item">55-56</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2010-09-29" who="CDS">Converted to September 2009 EpiDoc</change>
+            <change when="2011-05-13" who="EM">More tweaks for bib only inscriptions</change>
+            <change when="2019-11-18" who="GG">Adding transcription and data</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url=""/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/>θεῷ κισινω
+                <lb n="2"/>ὑπέρ σωτηρίας
+                <lb n="3"/><persName><name type="gentilicium" key="Aelius Αἴλιος">Αἰλίου</name> <name type="cognomen" key="Festus Φήστος">Φήστου</name>
+                <lb n="4"/><persName><name type="gentilicium" key="Pomponia Πομπώνια">Πομπωνίας</name>
+                <lb n="5"/>καὶ τῶν τέκνων
```

Comment:
Check the transcription -- something seems to have gone wrong here. Based on the ed. pr. I think something is missing from the name in l. 4, l. 7 I think is something different, and we want to double check that. (I'm relying on the the editions I have access to since I don't have ready access to an image.)